tests/test_1_location_to_square.py

```diff
@@ -0,0 +1,38 @@
+from src.qth_locator import location_to_square
+
+
+def test_location_to_square_positive():
+    squares = [
+        (None, (0, 0)),
+        (None, (32, 26)),
+        ("KM32jn", (32.541667, 26.75)),
+        ("KM32jn", (32.570831, 26.750003)),
+        ("KM72kk", (32.437487, 34.875015)),
+        ("JN45fo", (45.603333, 8.456667)),
+        ("KO92so", (52.601484, 39.565160)),
+        ("KM72jb", (32.071209, 34.780089)),
+        ("KM72jb", (32.07, 34.78)),
+    ]
+    for sq, loc in squares:
+        qth = location_to_square(loc[0], loc[1])
+        # print(f'{loc}: {sq}, calculated: {qth}, match:{sq==qth}')
```

Comment:
You should avoid debugging comments in final code, event test code. For feedback, you have test suite and its errors.

Reply:
Good catch 👍 Thanks 🙏 
Fixed in: https://github.com/Pytlicek/hamradio_functions/pull/8/commits/c50dc3120a26683bd073ad94f904699bee7e5120

Reply:
New fix here: https://github.com/Pytlicek/hamradio_functions/commit/ee6ee079dcf419d2210c6d534b41df59fb5419be